pyproject.toml

```diff
@@ -1,7 +1,13 @@
+[project]
+name = "nanobind"
+version = "2.0.1"
```

Comment:
The other info needs to be here. "hatch new --init" cam=n do the conversion for you.